### PR TITLE
feat: added comment to post

### PR DESCRIPTION
**Pull request by Shubhamkr2610**

### CODE REVIEW

File: src/components/posts/Posts.js [hunk 1]
```diff
@@ -105,6 +121,34 @@ export const Posts = ({
           </div>
         </div>
 
+           {/* ===============
+        comment input section 
+        =============== */}
```
Comment (Muzammil-khan710) [inline src/components/posts/Posts.js]: Remove commented code

File: src/components/posts/Posts.js [hunk 2]
```diff
@@ -127,7 +171,7 @@ export const Posts = ({
         {/* ===============
         modal for delete the post
         =============== */}
-
+          
         {/* ===============
         modal for edit the post
         =============== */}
```
Comment (Muzammil-khan710) [inline src/components/posts/Posts.js:201]: Remove commented code

**Review by Muzammil-khan710 — APPROVED** [review]
UI is clean
All actions are working fine
It's good to merge.

**Review by arjun-computer-geek — APPROVED** [review]
Codes are good

**Pull request merged**